### DISCUSSION TITLE
All album pictures have a vague (unclear) feeling

### DISCUSSION THREAD

**Discussion by Think2twice** [ciderapp/Cider#584] [Q&A] [answered]
I'm not sure if it's a 4K resolution display problem or something else

![image](https://user-images.githubusercontent.com/46246070/159025554-c1b1e1ed-1e1d-4a96-9a9d-76f89c5dcb42.png)

The album pictures are not as clear as Apple Music in MAC

It seems that the lyrics also have this feeling (fuzzy), which is still a little different from the original(Apple Music).
But if the software itself can only be like this, I think it's great.
![image](https://user-images.githubusercontent.com/46246070/159027248-c5577d21-612e-4308-8c69-6d75b5c108ad.png)

Without this vague feeling, I think this software is perfect and even better than the native software

**Comment by GamingLiamStudios** [comment, marked as answer]
The Album covers look not as sharp because the resolution of them is fixed & are just scaled to fit.
The lyrics are fuzzy because they aren't the current lyric being sung, so they are slightly blurred iirc.
